boards/arm/nrf54h20dk_nrf54h20/nrf54h20dk_nrf54h20-memory_map.dtsi

```diff
@@ -181,4 +181,18 @@
 			reg = <0x126000 DT_SIZE_K(64)>;
 		};
 	};
+
+	cpuapp_rw_partitions: cpuapp-rw-partitions {
+		compatible = "nordic,owned-partitions", "fixed-partitions";
+		status = "disabled";
```

Comment:
isn't default status disabled?

Reply:
It is enabled under app core description:
https://github.com/nrfconnect/sdk-zephyr/pull/1596/files#diff-2057cf121c362c3e459647d6ebc41a49efd464eff2713e2b6c1dc87612cce4f4R133